examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">blue, _, red = sns.color_palette()[:3]
</pre><p>The third colour from the palette is green, as can be seen in the plot...I guess this was meant to be</p><pre class="ql-syntax" spellcheck="false">blue, _, __, red = sns.color_palette()[:4]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
> From a quick look with a color blindness simulator, it looks like this plot is not very easy to interpret for people with tritanomaly. I think we should use the arviz-darkgrid theme and use C0 and C1 as colors here.

I've used these suggestions Oriol's instead, if it's ok.

Reply:
Yes, thanks, I'd missed that comment :flushed: !